manipulation/src/main/java/org/jboss/gm/manipulation/actions/MavenPublicationRepositoryAction.java

```diff
@@ -43,7 +44,9 @@
     @Override
     public void execute(Project project) {
         if (!project.getPluginManager().hasPlugin("maven")) {
-            logger.warn("Legacy 'maven' plugin not detected, skipping publishing repository creation.");
+            // This should never happen due to prior checks in ManipulationPlugin
+            throw new ManipulationUncheckedException(
+                    "Legacy 'maven' plugin not detected, skipping publishing repository creation.");
```

Comment:
@geoand  I don't think this is even possible given the checks in ManipulationPlugin. Potentially the entire block could be removed.

Reply:
Makes sense :ok_hand: